### ISSUE TITLE
ContextualMenu accessibility issue when navigating menu

### ISSUE THREAD

**Issue by alton-browsing** [microsoft/fluentui#15755]
### Environment Information

- **Package version(s)**: office-ui-fabric-react 7.115.3
- **Browser and OS versions**: Windows 10 Version 2004 (OS Build 20206.1000) Edge Dev Version 86.0.622.3

### Describe the issue:

Menu items are navigable with both 'Tab' and 'Arrow' key navigation but should only allow for the 'Arrow' key. Users with motor disability who rely on keyboard need to use more 'Tabs' to move out of the menu.

### Please provide a reproduction of the issue in a codepen:

Open the menu and reproduce the issue by pressing the 'Tab' key.

https://codepen.io/pen/?&editable=true=https%3A%2F%2Fdeveloper.microsoft.com%2F

#### Actual behavior:

Menu items are navigable with both 'Tab' and 'Arrow' key navigation.

#### Expected behavior:

Menu items should be navigable only with 'Arrow' key navigation.

**Comment by paulgildea** [comment]
@alton-browsing Thanks for filing this issue!

@smhigley Is this the right behavior? 

@joschect Any input from your side to see if this is a by design or not.
